go/client/logclient.go

```diff
@@ -71,7 +72,7 @@ type getSTHResponse struct {
 
 // getConsistencyProofResponse represents the JSON response to the CT get-consistency-proof method
```

Comment:
comment nit:  "to the get-consistency-proof CT method" would be consistent (ahem) with comments for other getFooResponse calls above.